app/app.go

```diff
@@ -395,9 +409,10 @@ func NewSifApp(
 	// CanWithdrawInvariant invariant.
 	app.mm.SetOrderBeginBlockers(
 		capabilitytypes.ModuleName,
+		upgradetypes.ModuleName,
```

Comment:
Any specific reason why we need change the order ? As in move upgrade up ? 
I cant foresee a problem with it , but the ordering has caused issues during upgrade before

Reply:
right, better to keep the order if it was working correctly